GameData/kappa-ray/kapparay_Squad_pods.cfg

```diff
@@ -107,3 +120,23 @@
 	%MODULE[ModuleKappaRayCrewPod] {
 	}
 }
+
+@PART[mk3CrewCabin]:FOR[kapparay]
+{
+	%MODULE[ModuleKappaRayAbsorber] { // shielding
+		absorpCoeff = 1.0 // the shuttle is state-of-the-art.  But some k-rads will still get through...
+	}
+	%MODULE[ModuleKappaRayCrewPod] {
+	}
+}
+
+
+//----------Misc
+@PART[seatExternalCmd]:FOR[kapparay]
```

Comment:
I'm not sure this part should have a ModuleKappaRayCrewPod.  When a Kerbal is in an external seat, I *think* he'll still be considered to be on EVA, and will thus still have an attached ModuleKappaRayEVA.  So unless you've tested it and found otherwise, I'd say best not to add modules to this part.  (I don't think the Absorber will be needed either, because I'd expect the collision box of the seat to be rather small and not cover the Kerbal much.  Where rads _do_ have to pass through the seat to get him, they probably should get the same slight attenuation they'd get from other parts.

But this really does need experiments to test how the game treats Kerbals in EVA seats.